src/components/TournamentBrackets.vue

```diff
@@ -0,0 +1,89 @@
+<script setup lang="ts">
+import type { TournamentDeref } from '@/models/tournament';
+import { useTournamentStore } from '@/stores/tournament.store';
+
+const props = defineProps<{
+  tournament: TournamentDeref;
+}>();
+
+const tournamentStore = useTournamentStore();
+const {
+  get_col_style,
+  get_matchs_per_round,
+  knockout_match_results,
+  get_winner_matchs_per_round,
+  get_looser_matchs,
+  get_validated_team_by_id,
+} = tournamentStore;
+</script>
+
+<template>
+  <section id="brackets">
+    <h1 v-if="props.tournament?.brackets?.length === 0" class="mt-6 text-center text-4xl">
+      Les arbres ne sont pas disponibles.
+    </h1>
+    <div v-for="bracket in props.tournament.brackets" v-else :key="bracket.id">
+      <h1 class="title">
+        {{ bracket.name }}
+      </h1>
+      <div v-if="bracket.bracket_type === BracketType.SINGLE" :key="bracket.id" class="overflow-x-auto">
```

Comment:
There are a lot of linting errors. This one is causing a crash because brackets can't be loaded.

Reply:
I fixed some of them but I will need your help for the others please.